app/_kong_plugins/datakit/examples/authenticate-with-vault-secret.yaml

```diff
@@ -0,0 +1,42 @@
+description: |
+  Authenticate to a third-party service using Vault secrets.
+
+extended_description: |
+  Authenticate to a third-party service using Vault secrets. Refer to the [Vault documentation](https://developer.konghq.com/gateway/entities/vault) for more information on how to set up and use Vault secrets.
+
+  This example contains the following nodes:
+
+title: Authenticate to a third-party service using Vault secrets.
+weight: 900
+
+config:
+  resources:
+    vault:
+      token: {vault://my-vault/my-token}
+  nodes:
+  - name: AUTH_HEADERS
+    type: jq
+    inputs:
+      token: vault.token
+    jq: |
+      {
+        "Authorization": "Bearer " + .apikey
+        "Content-Type": "application/json"
+      }
+
+  - name: AUTH_REQUEST
+    type: call
+    inputs:
+      heaqders: AUTH_HEADERS
```

Comment:
Typo here, and I think possibly on the jq node.

We don't have any automatic doc testing as of now so please double and triple check any of your changes and/or examples against a running instance of the gateway before merging.